catalogue/webapp/pages/download.js

```diff
@@ -153,18 +167,18 @@ const DownloadPage = ({ workId, sierraId, manifest, work }: Props) => {
   );
 };
 
-DownloadPage.getInitialProps = async (ctx: Context): Promise<Props> => {
+export const getServerSideProps = async (
+  ctx: Context
+): Promise<{ props: Props }> => {
```

Comment:
out of interest, was `Promise<Props>` incorrect, or is this just more explicit?

Reply:
A difference between `getInitialProps` and `getServerSideProps` is the return type i.e.
```ts
// getInitialProps
return { ...props }

// getServerSideProps
return { props: props }
```